client/src/BlockList/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useState } from "react";
+import { useQuery } from "@apollo/client";
+
+// blockList
+import BlockList from "BlockList/components/BlockList";
+import { QUERY_BLOCK_LIST } from "BlockList/query";
+
+// common
+import TableLoadingSkeleton from "common/components/TableLoadingSkeleton";
+
+const BlockListContainer: FC = () => {
+  const [page, setPage] = useState(0);
+  const PAGE_SIZE = 10;
+  const { data, error, loading } = useQuery(QUERY_BLOCK_LIST, {
+    variables: { limit: PAGE_SIZE, offset: page * PAGE_SIZE },
+  });
+
+  if (loading) {
+    return (
+      <div className="w-full">
```

Comment:
It seems we're using the same component in `ExtrinsicList/index.tsx`, maybe we can move it to `common/components`